internal/telemetry/collector_test.go

```diff
@@ -1129,6 +1129,60 @@ func TestCollectInstallationFlags(t *testing.T) {
 	}
 }
 
+func TestCollectBuildOS(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		name    string
+		buildOS string
+		wantOS  string
+	}{
+		{
```

Comment:
see previous question, what `wantOS` do we want if the `name` is an empty string?